java-datalineage/.OwlBot.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+deep-remove-regex:
+- "/java-datalineage/grpc-google-.*/src"
+- "/java-datalineage/proto-google-.*/src"
+- "/java-datalineage/google-.*/src"
+
+deep-preserve-regex:
+- "/java-datalineage/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
+
+deep-copy-regex:
+- source: "/google/cloud/datacatalog/lineage/(v.*)/.*-java/proto-google-.*/src"
+  dest: "/owl-bot-staging/java-datalineage/$1/proto-google-cloud-datalineage-$1/src"
+- source: "/google/cloud/datacatalog/lineage/(v.*)/.*-java/grpc-google-.*/src"
+  dest: "/owl-bot-staging/java-datalineage/$1/grpc-google-cloud-datalineage-$1/src"
+- source: "/google/cloud/datacatalog/lineage/(v.*)/.*-java/gapic-google-.*/src"
+  dest: "/owl-bot-staging/java-datalineage/$1/google-cloud-datalineage/src"
+
+
+api-name: java-datalineage
```

Comment:
This shouldn't have "java-" prefix. Where did it come from?

https://github.com/googleapis/google-cloud-java/blob/main/java-aiplatform/.OwlBot.yaml#L30

Reply:
I believe it's coming from: https://github.com/googleapis/google-cloud-java/blob/main/generation/new_client/new-client.py#L239 and https://github.com/googleapis/google-cloud-java/blob/main/generation/new_client/templates/owlbot.yaml.monorepo.j2#L33

What is the api-name in this .OwlBot.yaml file used for?

Reply:
go/owlbot-howto

> api-name: Gets inserted into pull request titles. Useful for mono-repos.

Reply:
Thanks. I've manually updated for this PR.